iis-wf/iis-wf-primary/src/main/resources/eu/dnetlib/iis/wf/primary/main/oozie_app/workflow.xml

```diff
@@ -490,6 +490,10 @@
                     <name>output_faults</name>
                     <value>${workingDir}/primary_import/faults</value>
                 </property>
+                <property>
+                    <name>output_report</name>
+                    <value>${workingDir}/report</value>
```

Comment:
To me your proposition for joining of partial reports directly in parent workflow is also complicated more then it needs to be. This way we would end up with extra joining node in every middle workflow.